From: Darik Horn <dajhorn@vanadac.com>
Date: Sat, 18 Oct 2014 19:48:46 -0500
Subject: Add libuutil to LIBADD for libzfs and libzfs_core

Resolve spurious dpkg-shlibdeps warnings like this:

	warning: symbol strlcpy used by libzfs_core.so found in none of the libraries

This happens because dpkg-shlibdeps does not recursively search for symbols.

Signed-off-by: Darik Horn <dajhorn@vanadac.com>
---
 lib/libzfs/Makefile.am      | 1 +
 lib/libzfs_core/Makefile.am | 3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

Index: b/lib/libzfs/Makefile.am
===================================================================
--- a/lib/libzfs/Makefile.am
+++ b/lib/libzfs/Makefile.am
@@ -27,6 +27,7 @@
 
 libzfs_la_LIBADD = \
 	$(top_builddir)/lib/libzfs_core/libzfs_core.la \
+	$(top_builddir)/lib/libuutil/libuutil.la \
 	$(top_builddir)/lib/libshare/libshare.la \
 	$(top_builddir)/lib/libnvpair/libnvpair.la \
 	$(top_builddir)/lib/libzpool/libzpool.la
Index: b/lib/libzfs_core/Makefile.am
===================================================================
--- a/lib/libzfs_core/Makefile.am
+++ b/lib/libzfs_core/Makefile.am
@@ -10,6 +10,7 @@
 	$(top_srcdir)/lib/libzfs_core/libzfs_core.c
 
 libzfs_core_la_LIBADD = \
-	$(top_builddir)/lib/libnvpair/libnvpair.la
+	$(top_builddir)/lib/libnvpair/libnvpair.la \
+	$(top_builddir)/lib/libuutil/libuutil.la
 
 libzfs_core_la_LDFLAGS = -version-info 1:0:0
